metaflow/plugins/cards/card_decorator.py

```diff
@@ -66,16 +65,6 @@ def _load_card_package(self):
             file_path, arcname = path_tuple
             yield (file_path, os.path.join("metaflow", "plugins", "cards", arcname))
```

Comment:
@romain-intel : you mean these too right ?

Reply:
Yep, I think all packaging should now be taken care of (the metaflow packaging includes the .html, .js, .css by default). I think going forward, we should probably evolve the `add_to_package` function to return the extensions that it wants packaged or something like that.